lib/va_profile/disability/service.rb

```diff
@@ -37,7 +33,7 @@ def get_disability_data
         error_status = e&.status
         if error_status == 404
           log_exception_to_sentry(
-            e, { edipi: @edipi }, { va_profile: :disability_rating_not_found }, :warning
+            e, { edipi: @user.edipi }, { va_profile: :disability_rating_not_found }, :warning
```

Comment:
This should not be @user.edipi. This should remain @edipi

Reply:
@RachalCassity we decided to pass in the full user and not the edipi in `lib/va_profile/prefill/military_information.rb`:
 `@disability_service = VAProfile::Disability::Service.new(user)`